qiskit/pulse/pulse_lib/discrete.py

```diff
@@ -102,16 +142,35 @@ def sawtooth(duration: int, amp: complex, period: float = None,
 
 def triangle(duration: int, amp: complex, period: float = None,
              phase: float = 0, name: Optional[str] = None) -> 'SamplePulse':
-    """Generates triangle wave `SamplePulse`.
+    r"""Generates triangle wave :class:`~qiskit.pulse.SamplePulse`.
+
+    For :math:`A=` ``amp``, :math:`T=` ``period``, and :math:`\phi=` ``phase``,
+    applies the `midpoint` sampling strategy to generate a discrete pulse sampled from
+    the continuous function:
+
+    .. math::
 
-    Applies `midpoint` sampling strategy to generate discrete pulse from continuous function.
+        f(x) = A \left(-2\left|\text{sawtooth}(x, A, T, \phi)\right| + 1\right)
+
+    This corresponds to a sine wave with linear ramps.
```

Comment:
I don't think this is a "sine wave", if anything it is something like

```suggestion
    This a non-sinusoidal wave with linear ramping.
```